TSC-Charter.md

```diff
@@ -129,41 +117,37 @@ of terms a TSC Chairperson or voting CPC member may serve.
 ## Section 7. Voting
 
 For internal project decisions, Collaborators shall operate under Lazy
-Consensus. The TSC shall establish appropriate guidelines for
+Consensus. The TSC voting members shall establish appropriate guidelines for
 implementing Lazy Consensus (e.g. expected notification and review time
 periods) within the development process.
 
-The TSC follows a [Consensus Seeking][] decision making model. When an agenda
-item has appeared to reach a consensus the moderator will ask "Does anyone
-object?" as a final call for dissent from the consensus.
-
-If an agenda item cannot reach a consensus a TSC member can call for
-either a closing vote or a vote to table the issue to the next meeting.
-The call for a vote must be seconded by a majority of the TSC or else the
-discussion will continue.
```

Comment:
Why is this dropped as a part of this PR? Is it to remove this tabling "vote" from the count for voting/regular member status?

Reply:
I removed it because I have been on the TSC for I-don't-know-how-many years and this process has never once been used. So I wanted to take this opportunity to simplify the charter so that it reflects our actual processes. However, I can put it back in and remove it in a separate proposal if you prefer.

Reply:
I have no preference on this, really just wanted an explicit comment/explanation like this available somewhere.